neon_core/configuration/neon.yaml

```diff
@@ -10,10 +10,12 @@ audio_parsers:
   blacklist:
   - gender
 stt:
-  module: deepspeech_stream_local
-  fallback_module: google_cloud_streaming
+  module: ovos-stt-plugin-server
+  fallback_module: deepspeech_stream_local
+  ovos-stt-plugin-vosk:
+    model: /home/neon/.local/share/neon/vosk-model-small-en-us-0.15
   ovos-stt-plugin-server:
-    url: https://stt.openvoiceos.com/stt
+    url: https://whisper.neonaiservices.com/stt
```

Comment:
consider adding it to the public server list https://github.com/OpenVoiceOS/ovos-stt-plugin-server/blob/dev/ovos_stt_plugin_server/__init__.py#L25

and uptime status page https://github.com/OpenVoiceOS/status

Reply:
https://github.com/OpenVoiceOS/ovos-stt-plugin-server/pull/20